assets/src/scripts/charcoal/admin/widget/form.js

```diff
@@ -50,6 +50,7 @@ Charcoal.Admin.Widget_Form.prototype.set_properties = function (opts) {
     this.group_conditions = opts.data.group_conditions;
     this.$form            = $(this.form_selector);
     this.allow_reload     = opts.data.allow_reload;
+    this.force_refresh    = opts.data.force_refresh;
```

Comment:
I think this property should be called `force_page_reload` to clarify its purpose, match the [`Location.reload()` API](https://developer.mozilla.org/en-US/docs/Web/API/Location/reload), and distinguish itself from `allow_reload`, which should have been called `force_widget_reload`.

```suggestion
    this.force_page_reload = opts.data.force_page_reload;
```